Assets/Scripts/Enemies/SentryTrigger.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SentryTrigger : MonoBehaviour
+{
+    private void OnTriggerEnter(Collider collider)
+    {
+        if (collider.gameObject.name == "GameDollyCart")
+        {
+            transform.parent.GetComponent<Sentry>().enabled = true;
+            transform.parent.GetComponent<Sentry>().SetPlayer(collider.transform.GetChild(1).gameObject);
```

Comment:
I Feel like this is a little dangerous to do if we ever reorganize the player prefab, I would changed this to transform.parent.GetComponent<Sentry>().SetPlayer(collider.GetComponentInChildren<PlayerHealth>().gameObject);, but also i feel like the playerHealth should probably be turned to a singleton to make this easier